### PR TITLE
pr test

### PR DESCRIPTION
**Pull request by JeremBe**
pr test - edit

### CODE REVIEW

File: app/app.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ const server = http.createServer(app)
 server.listen(port)
 
 server.on('listening', () => {
-  console.log(`Server listening on port ${port}`)
+  console.log(`server listening on port ${port}`)
```
Comment (JeremBe) [inline app/app.ts:15]: test

File: app/app.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ const server = http.createServer(app)
 server.listen(port)
 
 server.on('listening', () => {
```
Comment (JeremBe) [inline app/app.ts:14]: test

**Review by JeremBe — COMMENTED** [review]
comment

**Review by thibautrey — COMMENTED** [review]
These changes are fundamentally important. I agree to them. Let's change the word!

**Pull request closed without merging**